packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -767,6 +767,16 @@ const CommentBoard = ( {
 			? actions.filter( ( item ) => item.isEligible( thread ) )
 			: [];
 
+	let deleteConfirmMessage = __(
+		'Are you sure you want to delete this note?'
+	);
+
+	// When deleting a top level note, descendants will also be deleted.
+	if ( canResolve ) {
```

Comment:
```suggestion
	if ( thread.parent === 0 ) {
```

Probably better to use parent check directly. The `canResolve` value might change in future.